hypothesis-python/tox.ini

```diff
@@ -135,19 +135,19 @@ commands =
 commands =
     pip install .[pytz]
     pip install django~=3.2.15
-    python -bb -X dev -m tests.django.manage test tests.django
+    python -bb -X dev -m tests.django.manage test tests.django {posargs}
 
 [testenv:django41]
 commands =
     pip install django~=4.1.0
-    python -bb -X dev -m tests.django.manage test tests.django
+    python -bb -X dev -m tests.django.manage test tests.django {posargs}
 
 [testenv:django42]
 setenv=
     PYTHONWARNDEFAULTENCODING=1
 commands =
     pip install django~=4.2.0
-    python -bb -X dev -m tests.django.manage test tests.django
+    python -bb -X dev -m tests.django.manage test tests.django {posargs}
```

Comment:
I tested with Django 5.0 as well, but that raised enough warnings that it would be worth addressing in its own PR.